doc/access.md

```diff
@@ -0,0 +1,64 @@
+# Access to Node.js Infrastructure
+
+Documents which groups have access to which Infra assets. Note that links to
+`@nodejs/` teams are not visible to people who aren't in the Nodejs
+organisation, so those links may not work for you. The [secrets repo][] is also
+secret...
+
+## Machine Access
+
+For a list of machines, see the [inventory.yml][]. Secrets are stored in the
+[secrets repo][], which [@nodejs/build][] (and [org owners][]) have access to.
+Secrets are individually encrypted, so access to the repo does not itself
+give access to any of the secrets within. For more info see the repo's README.
+
+### Test machines
+
+[@nodejs/build][] have root access to the test CI machines (`test-*`).
+
+### Infra and Release machines
+
+A subsection of build members have access to infra and release machines
+(`infra-*` and `release-*`). The current list is:
```

Comment:
I need to add a note to clarify that this includes `ci.nodejs.org`, so any CI issues that can't be resolved within Jenkins will have to be resolved by this group.